pkgs/tools/misc/agedu/default.nix

```
@@ -1,15 +1,15 @@
 {stdenv, fetchgit, autoreconfHook, halibut}:
 let
-  date = "20180918";
-  rev = "80c7beb";
+  date = "20200206";
+  rev = "963bc9d";
 in
 stdenv.mkDerivation {
   name = "agedu-${date}.${rev}";
```

```suggestion
  pname = "agedu";
  version = "unstable-2020-02-06";
```
is more standard I think.

---

No it's the official versioning scheme: see https://www.chiark.greenend.org.uk/~sgtatham/agedu/.